aspnetcore/fundamentals/static-files/samples/9.x/StaticFilesSample/Program.cs

```diff
@@ -383,18 +385,18 @@
 app.UseHttpsRedirection();
 
 // <snippet_mul>
-app.UseStaticFiles(); // Serve files from wwwroot
-app.UseStaticFiles(new StaticFileOptions
-{
-    FileProvider = new PhysicalFileProvider(
-        Path.Combine(builder.Environment.ContentRootPath, "MyStaticFiles"))
-});
+ app.UseStaticFiles();
+ app.UseStaticFiles(new StaticFileOptions
+ {
+     FileProvider = new PhysicalFileProvider(
+         Path.Combine(builder.Environment.ContentRootPath, "MyStaticFiles"))
+ });
```

Comment:
@javiercn can you provide a `app.MapStaticAssets(staticAssetsManifestPath: "MyManifest");` and the format of the manifest file?
Take a look at all the `app.UseStaticFiles(new StaticFileOptions` occurances, I think this is the only one that can be replaced with `app.MapStaticAssets(staticAssetsManifestPath: "MyManifest");`